evalml/tests/automl_tests/test_automl.py

```diff
@@ -1715,3 +1715,15 @@ def test_automl_error_callback(mock_fit, mock_score, X_y_binary, caplog):
     assert len(automl._results['errors']) == 1  # Raises exception at first error
     for e in automl._results['errors']:
         assert str(e) == msg
+
+
+@patch('evalml.pipelines.BinaryClassificationPipeline.score')
+@patch('evalml.pipelines.BinaryClassificationPipeline.fit')
+def test_automl_woodwork_passed_to_pipelines(mock_fit, mock_score, dummy_binary_pipeline_class, X_y_binary):
```

Comment:
Test that checks that woodwork is passed to pipelines :)

Reply:
Do we check anywhere that passing in woodwork does not result in a conversion and that user-defined types are preserved throughout? Might be worth to do this on automl search on small scale: the only pipeline allowed uses mock components, and we only do two iterations.

Reply:
Yup, added `test_automl_woodwork_user_types_preserved` for this :D